sync/src/test/java/tech/pegasys/artemis/sync/PeerSyncTest.java

```diff
@@ -375,6 +380,11 @@ void sync_handleEmptyResponse() {
     // Check that the sync is done and the peer was not disconnected.
     assertThat(syncFuture).isCompleted();
     verify(peer, never()).sendGoodbye(any());
+
+    // check startingSlot
+    UnsignedLong startingSlot = peerSync.getStartingSlot();
+    assertThat(startingSlot).isGreaterThan(UnsignedLong.valueOf(0));
+    assertThat(startingSlot).isLessThan(PEER_HEAD_SLOT);
```

Comment:
I'd have expected we could expect specific values here as well.

Reply:
done